Docs: corrected spelling

DikaCream opened the pull request:
The text contains a typo in "tranasction", which should be corrected to "transaction"

jarrodwatts commented:
thanks for this @kasrakhosravi. Currently, we're not going to add specific integrations like this to our core docs, however.

We will need a place for developer integrations like this outside of the core documentation in the near future

cygaar commented:
This is a fantastic writeup though, much appreciated!

kasrakhosravi commented:
@jarrodwatts @cygaar totally understand and thank you. would be very interested to get your personal feedback on this tool, if you ever got the chance to give it a try

DikaCream committed:
Docs: corrected spelling

The text contains a typo in "tranasction", which should be corrected to "transaction"

jarrodwatts commented:
thanks @DikaCream
